templates/create-daml-app/daml/Setup.daml

```diff
@@ -3,45 +3,63 @@ module Setup where
 import DA.Foldable (forA_)
 import DA.Optional (fromSomeNote)
 import qualified DA.Text as T
+import qualified DA.TextMap as M
 import Daml.Script
 
 -- | A test user for the create-daml-app network.
 data TestUser = TestUser with
   alias : Text
   public : Party
+  participantAlias: Text
+
+data ParticipantDatails = ParticipantDatails with
```

Comment:
Typo? ParticipantD**e**tails

Reply:
yeah, we don’t expect to merge this though so doesn’t really matter.